exchanges/retry/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@urql/exchange-retry",
+  "version": "1.8.4",
```

Comment:
You may want to start at either `0.1.0` or `1.0.0` and adjust the changelog to remove that `1.8.4` entry :)